cesium/featureset.py

```diff
@@ -47,12 +47,16 @@ def impute(self, strategy='constant', value=None):
             `None`, in which case a very large negative value is used (a
             good choice for e.g. random forests).
 
+        max_value : float, optional
+            Maximum (absolute) value above which values are treated as infinite.
```

Comment:
What is the rationalization for this parameter?  (I guess it is related to the scikit-learn discussion we had the other day, but may be worth outlining here in some more detail.)